runtime/moonbase/src/lib.rs

```diff
@@ -348,8 +351,11 @@ where
 		mut fees_then_tips: impl Iterator<Item = Credit<R::AccountId, pallet_balances::Pallet<R>>>,
 	) {
 		if let Some(fees) = fees_then_tips.next() {
-			// for fees, 80% are burned, 20% to the treasury
-			let (_, to_treasury) = fees.ration(80, 20);
+			let treasury_perbill =
+				runtime_params::dynamic_params::runtime_config::FeesTreasuryPercentage::get();
```

Comment:
```suggestion
				runtime_params::dynamic_params::runtime_config::FeesTreasuryProportion::get();
```

If it's not a percentage, please don't name it percentage